runners/apex/src/main/java/org/apache/beam/runners/apex/ApexPipelineOptions.java

```diff
@@ -32,12 +32,6 @@
 
   String getApplicationName();
 
-  @Description("set parallelism for Apex runner")
-  void setParallelism(int parallelism);
```

Comment:
I think you do want to keep this, don't you? It is unfortunate that we have this global options namespace, but that is all about the ability to make it command-line friendly. I think perhaps you should match the Flink value exactly, or (better) factor this into some common `Options` class.